src/env-inl.h

```diff
@@ -468,6 +468,14 @@ inline void Environment::set_trace_sync_io(bool value) {
   options_->trace_sync_io = value;
 }
 
+inline bool Environment::get_still_synchronous() {
+  return still_synchronous_;
+}
+
+inline void Environment::set_still_synchronous(bool value) {
+  still_synchronous_ = value;
+}
+
```

Comment:
Adding these shouldn't be necessary. Using `perf_hooks.performance.nodeTiming.loopStart` should be sufficient.

Reply:
Yeah, that works perfectly, thanks!